infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -222,10 +214,10 @@ public TokenType txnCanCommit(TxResolutionInfo txInfo, /** Input. */
         log.trace("Commit-req[{}]", txInfo);
         final long txSnapshotTimestamp = txInfo.getSnapshotTimestamp();
 
-        if (txSnapshotTimestamp < globalLogStart.get() - 1) {
-            log.debug("ABORT[{}] snapshot-ts[{}] failover-ts[{}]",
-                    txSnapshotTimestamp, globalLogStart.get());
-            return TokenType.TX_ABORT_NEWSEQ;
+        if (txSnapshotTimestamp < maxConflictWildcard) {
```

Comment:
this will cause false aborts unnecessarily  - check if there is information about the conflict key, and only if null, then compare with wildcard!